resources/default-settings/jukebox.default.yaml

```diff
@@ -77,6 +77,9 @@ jinglemp3:
 alsawave:
   # Config of the Wave through ALSA Jingle Service
   device: default
+players:
+  content:
+    audiofile: /../../shared/audiofolders/audiofiles.yaml
```

Comment:
What is this supposed to do again? This change does not include a example of this file

Reply:
> Within this file the user can specify his files that should be visible to play.

You mean just for Spotify? Or also for MP3? (For MP3, this would be an anti-pattern as MPD already takes care of catalogging)

Reply:
> You mean just for Spotify? Or also for MP3? (For MP3, this would be an anti-pattern as MPD already takes care of catalogging)

The problem here is that you need a manager who knows everything about both worlds and can activate the corresponding player based on the content.

Another option would be to ask every available player for their known content...